cool-core/src/main/java/com/nus/cool/core/cohort/ExtendedCohortSelection.java

```diff
@@ -456,16 +485,16 @@ private int getUserBirthTime(int start, int end) {
 			birthOffset = (birthOffset < offset) ? offset : birthOffset;
 		}
 
-		for (Integer e : sortedEvents) {
-			// filter between [start, birthOffset] for birth events
-			// without time window
-			if (seq.getBirthEvents().get(e).getTimeWindow() == null) {
-				offset = (eventOffset.get(e).isEmpty()) ? start : eventOffset.get(e).getLast() + 1;
-				filterEvent(e, offset, birthOffset);
-				if (!checkOccurrence(e))
-					return -1;
-			}
-		}
+//		for (Integer e : sortedEvents) {
+//			// filter between [start, birthOffset] for birth events
```

Comment:
Also there are many places with commented out codes, Safe to remove them?

Reply:
We will remove them at last. Just not sure whether the logic is correct.